static/app/views/dashboardsV2/widgetBuilder/groupBy.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'sentry/components/button';
+import Field from 'sentry/components/forms/field';
+import {IconAdd, IconDelete} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {defined} from 'sentry/utils';
+import {QueryFieldValue} from 'sentry/utils/discover/fields';
+import {FieldValueOption, QueryField} from 'sentry/views/eventsV2/table/queryField';
+import {FieldValueKind} from 'sentry/views/eventsV2/table/types';
+import {generateFieldOptions} from 'sentry/views/eventsV2/utils';
+
+const GROUP_BY_LIMIT = 20;
+const EMPTY_FIELD: QueryFieldValue = {kind: FieldValueKind.FIELD, field: ''};
+
+type Props = {
```

Comment:
```suggestion
interface Props  {
```

not sure now if we are going to move to interfaces or keep the types, but just to be consistent with the other widget builder files